index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ahmad Zaki Yousufi Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <header class="header">
+        <nav class="statusbar">
+            <img class="logo" alt="My logo picture" src="css/image/logo/5309769.png">
+            <img class="menu" alt="Menu bar" src="css/image/Bars/Menu.svg">
+        </nav>
+    </header>
+    <div class="page1-content blackpage">
+        <h1 class="my-name">Hey There.<br>I'm Zaki</h1>
+        <h2 class="my-position">I’am a Software Developer</h2>
+        <p class="discription-me">I am a software developer a graphic designer and a video journalist.
+            I can help you build a product, feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need to be coded,
+            don’t hesitate to contact me. Also, I can help you in the field of videography, photography, and graphics.
+        </p>
+        <div class="page-footer">
+            <div class="social-media">
+                <a href="https://github.com/zakiyousufi" target="_blank" rel="noopener">
+                    <img class="github icons" alt="Github Icon" src="css/Icons/github.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="linkedin icons" alt="LinkedIn Icon" src="css/Icons/linkedin.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="due icons" alt="Social Media Icon" src="css/Icons/due.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="tweeter icons" alt="Tweeter Icon" src="css/Icons/tweeter.svg">
+                </a>
+                <a href="#" target="_blank" rel="noopener">
+                    <img class="m-icon icons" alt="Social Media Icon" src="css/Icons/M.svg">
+                </a>
+            </div>
+            <div>
```

Comment:
**[REQUIRED  CHANGES]**

- To enforce the use of HTML5 semantic tag, please kindly use a `<ul>` tag to wrap your list items like this:

```
<ul>
  <li>
    <a href="">
      <img src="" alt="">
    </a>
  </li>
</ul>

```